R/infrastructure.R

```diff
@@ -291,6 +291,27 @@ use_package_doc <- function(pkg = ".") {
   writeLines(out, file.path(pkg$path, path))
 }
 
+# help function for use_package
+# takes the same arguments as use_package
+build_package_txt <- function(package, version = NULL,
+                              compare =  c(">=", ">", "==", "<=", "<")) {
```

Comment:
I read that, but I disagree. I don't there's any point in exposing an interface that people are unlikely to use. But if you can find a couple of packages in the wild that use `<` or `==`, I might change my mind.